package.json

```diff
@@ -67,15 +67,17 @@
   },
   "resolutionsPurpose": "This temporarily chooses the right versions of these libs, which are specified in both deck AND deck-kayenta",
   "resolutions": {
+    "@types/angular": "1.6.26",
     "@uirouter/react-hybrid": "1.0.2",
     "@uirouter/react": "1.0.7",
     "@uirouter/angularjs": "1.0.26",
     "@uirouter/core": "6.0.8",
+    "angular": "1.8.0",
+    "cheerio": "=1.0.0-rc.6",
+    "nth-check": "2.0.1",
```

Comment:
```suggestion
    "nth-check": "2.1.1",
```

Reply:
I settled on `2.0.1` because `nth-check` isn't a direct dependency of the spinnaker/deck packages, but the highest transitive dependency is `2.0.0` where `2.0.1` fixes a vulnerability. 

The `2.1.1` release includes dual module CommonJS and ESM and I wasn't sure if I wanted to mess with that. :/

Reply:
Makes sense!